lib/user_agent/browsers/apple_watch/atc.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class UserAgent
+  module Browsers
+    module AppleWatch
+      ##
+      # This class detects and handles the ATC browser on Apple Watch
+      class ATC < Base
+        # The browser string
+        BROWSER = 'atc'
```

Comment:
Just did some independent research by mounting an iOS firmware image and poking around plists and strings. `atc` is used by `AirTraffic.framework` and so I think making `browser` return `"AirTraffic"` is more appropriate